.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,26 @@
+on:
```

Comment:
did you mean to commit this and the `knope.toml` in this PR?

Reply:
I did